AdminUi/src/AdminUi/ClientApp/src/app/app.module.ts

```diff
@@ -75,7 +76,8 @@ import { XSRFInterceptor } from "./shared/interceptors/xsrf.interceptor";
         ConfirmationDialogComponent,
         LoginComponent,
         ChangeSecretDialogComponent,
-        IdentitiesOverviewComponent
+        IdentitiesOverviewComponent,
+        IdentityEditRelationshipsComponent
```

Comment:
Why is it called **Edit**Relationships? We don't want to edit them. It's a readonly view.

Reply:
The file structure in the AdminUi for Identities is currently like this:
![image](https://github.com/nmshd/backbone/assets/118175875/eab2db48-b89e-4663-ba96-8d9753b16cd6)
So name would be better interpreted like IdentityEditComponent => RelationshipsComponent. As the RelationshipsComponent is part of the IdentityEditComponent.

Maybe we should instead rename the IdentityEditComponent to IdentityDetailsComponent, that way IdentityDetailsRelationships wouldn't look out of place.

Reply:
I think I like this more, yes. In fact, this would apply to the other edit-components as well. In my opinion, this is much more fitting, because depending on where you are, there aren't even "edit" capabilities in this view. 

But you don't need to change all names right now. We can do this in another PR. Just the identity one is okay for now.